app/javascript/spec/ansibleCatalog/catalogActions.test.js

```diff
@@ -0,0 +1,133 @@
+import fetchMock from 'fetch-mock';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import * as actions from '../../react/ansibleCatalog/catalogActions';
+import * as fixtures from './test.fixtures';
+
+const mockRequest = (url, response) => {
+  fetchMock
+    .getOnce(url, response);
+};
+const resources = [{ id: 1, name: 'test' }];
+describe('Catalog Actions', () => {
+  let mockStore;
+  let middlewares;
+  let store;
+  beforeEach(() => {
+    middlewares = [thunk];
+    mockStore = configureMockStore(middlewares);
+    store = mockStore({});
+  });
+  afterEach(() => {
+    fetchMock.reset();
+    fetchMock.restore();
+  });
+
+  it('displays a list of catalogs', () => {
+    const sampleAllCatalogs = [
+      ['test', 1],
+      ['test 2', 2],
+    ];
+
+    store.dispatch(actions.loadCatalogs(sampleAllCatalogs));
+    const displayCatalogActions = store.getActions();
+    expect(displayCatalogActions).toEqual([fixtures.catalogActionCatalogs]);
+  });
+  it('Fails a http request', () => {
+    // eslint-disable-next-line no-console
+    console.error = jest.fn();
+    const spy = jest.spyOn(window, 'add_flash');
+    window.add_flash = spy;
```

Comment:
this is weird, `jest.spyOn` should be modifying `window.add_flash`, so this assignment doesn't make sense to me